src/application/deployer/push_application_images.go

```diff
@@ -6,13 +6,13 @@ import (
 	"github.com/Originate/exosphere/src/aws"
 	"github.com/Originate/exosphere/src/docker/composebuilder"
 	"github.com/Originate/exosphere/src/docker/tools"
-	"github.com/Originate/exosphere/src/types"
+	"github.com/Originate/exosphere/src/types/deploy"
```

Comment:
Weren't we putting these in `application/deployer/types`? Or is that not what you meant?

Reply:
That was what I originally thought but having packages with the same name prevents goimports from working and thus would have to deal with imports in a more manual way.